planner/core/common_plans.go

```diff
@@ -271,6 +271,9 @@ func (e *Execute) getPhysicalPlan(ctx context.Context, sctx sessionctx.Context,
 		// type from "paramMarker" to "Constant".When Point Select queries are executed,
 		// the expression in the where condition will not be evaluated,
 		// so you don't need to consider whether prepared.useCache is enabled.
+		note := errors.New("Hit plan cache: True")
```

Comment:
how about use "errors" instead of "github.com/pingcap/errors" to generate this error?

pingcap/errors#New will fetch call `runtime.Caller` that maybe affect performance~